lib/scan/version.rb

```diff
@@ -1,4 +1,4 @@
 module Scan
-  VERSION = "0.4.1"
+  VERSION = "0.4.2"
```

Comment:
Locking the version of `rubocop` that doesn't require these changes. Added a commit @i2amsam